docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml

```diff
@@ -2,16 +2,16 @@ apiVersion: console.openshift.io/v1
 kind: QuickStarts
 metadata:
   name: openshift-ai-rhods-fasttrack
-  externalDocumentation: true
+  otherContentType: true
 spec:
   version: 0.1
   type:
-    text: Documentation
+    text: Other content type
     color: orange
-  displayName: FastTrack to Red Hat OpenShift Data Science
+  displayName: FastTrack to Red Hat OpenShift AI
   icon: >-
```

Comment:
@Hyperkid123 Should I remove the icon values, or specify them in some other way? The images are not shown on the cards in the UI?

Reply:
You can just remove it. It was a UX decision not to show them anymore.